web/src/main/webapp/my-app/layout/controllers.js

```diff
@@ -239,7 +329,7 @@ define(['angular', 'jquery'], function(angular, $) {
       /**
        * Sets href attribute for 'BASIC' type widget
        * @param portlet
-       * @returns {*}
```

Comment:
https://github.com/gajus/eslint-plugin-jsdoc recommends `returns`. Either `return` or `returns` works for me.
Either way we should add that to the lint rules and make it consistent across the app.

Reply:
It's in there already and triggers a warning. The instances changed in this PR were fixed automatically by running eslint with the --fix flag.